Source/SpatialGDK/Private/SpatialGameInstance.cpp

```diff
@@ -62,8 +62,11 @@ bool USpatialGameInstance::StartGameInstance_SpatialGDKClient(FString& Error)
 		GetEngine()->ShutdownWorldNetDriver(GetWorldContext()->World());
 	}
 
-	FURL BaseURL = WorldContext->LastURL;
-	FURL URL(&BaseURL, TEXT("127.0.0.1"), ETravelType::TRAVEL_Absolute);
+	UEditorEngine* const EditorEngine = CastChecked<UEditorEngine>(GetEngine());
```

Comment:
Is this used? Looks like it can be removed.

Reply:
Oh good catch.